submissions/QuickPerformance/QuickPerformance.md

```diff
@@ -0,0 +1,84 @@
+
+### What to do about all of that?
+
+There are a few causes that I mentioned in the last paragraph, such as switching from nuclear to solar, that you can do yourself. However, some things just can't be done by you and must be done with commands. So, here is a selection of the ones that I thought were the most useful for uncontrollable things, such as pollution.
+
+If you have a lot of pollution in places and you are further into the game where biters are just an expansion nuisance, you can run this command:
+
+```lua
+/c for _, surface in pairs(game.surfaces) do
+  surface.clear_pollution()
+end
+game.map_settings.pollution.enabled = false
+```
+
+For every surface in the game, it will remove all pollution and after that, it will disable pollution creation, which can improve performance quite a lot in some cases.
+
+If you however believe that pollution is not the right thing for you and instead you have large amounts of biters that are just idle, you can run this command instead:
+
+```lua
+/c game.forces["enemy"].kill_all_units()
+```
+
+It will kill all biters on the `"enemy"` force (which without any modifications will be biters). It will however **only remove mobile units, such as biters and spitters, not spawners or turrets**. If you would like to remove those too, you can use this command instead:
+
+```lua
+/c local surface=game.player.surface
+for key, entity in pairs(surface.find_entities_filtered({force="enemy"})) do
+	entity.destroy()
+end
+```
+
+If you *still* feel that biters are just annoying and your base always just deals with them without any input, you can use this command instead:
+
+```lua
+/c local surface = game.player.surface
+local mgs = surface.map_gen_settings
+mgs.autoplace_controls["enemy-base"].size = "none"
+surface.map_gen_settings = mgs
+```
+
+It will disable biters from generating **only on your surface**, which by default is *Nauvis*, however it won't affect current biters (you will need to use previous commands for that).
+
+If none of these work, the easiest resort to go to is reducing the game speed. Everything will run slower, but you will be able to play with your friends without them getting kicked off:
+
+```lua
+/c game.speed = 0
+```
+
+All of these have been sourced from the [Commands wiki page](https://wiki.factorio.com/Console) if you want to see more Lua magic. To run any command, simply copy it, open your chatbar and paste it in.
+
```

Comment:
It feels like there needs to be a bit of a conclusion here. Nothing too long but something about how optimized factorio is by design maybe? Also maybe mention the intentional choice not to talk about GPU bottlenecks in this piece too?